misc/vim/plugin/nit.vim

```diff
@@ -243,13 +243,61 @@ fun NitOmnifunc(findstart, base)
 		if !handled
 			" It may be a keyword
 			if strlen(a:base) > 0
-				for keyword in ['module', 'import', 'class', 'abstract', 'interface',
-					\'universal', 'enum', 'end', 'fun', 'type', 'init', 'redef', 'is',
-					\'do', 'var', 'extern', 'public', 'protected', 'private', 'intrude',
-					\'if', 'then', 'else', 'while', 'loop', 'for', 'in', 'and', 'or',
-					\'not', 'implies', 'return', 'continue', 'break', 'abort', 'assert',
-					\'new', 'isa', 'once', 'super', 'self', 'true', 'false', 'null',
-					\'as', 'nullable', 'isset', 'label']
+				for keyword in [
+					\'abort',
+					\'abstract',
+					\'and',
+					\'as',
+					\'assert',
+					\'break',
+					\'catch',
+					\'class',
+					\'continue',
+					\'__debug__',
+					\'do',
+					\'else',
+					\'end',
+					\'enum',
+					\'extern',
+					\'false',
+					\'for',
+					\'fun',
+					\'if',
+					\'implies',
+					\'import',
+					\'in',
+					\'init',
+					\'interface',
+					\'intrude',
+					\'is',
+					\'isa',
+					\'isset',
+					\'label',
+					\'loop',
+					\'module',
+					\'new',
+					\'not',
+					\'null',
+					\'nullable',
+					\'once',
+					\'or',
+					\'package',
+					\'private',
+					\'protected',
+					\'public',
+					\'redef',
+					\'return',
+					\'self',
+					\'subset',
+					\'super',
+					\'then',
+					\'true',
+					\'type',
+					\'universal',
+					\'var',
+					\'while',
+					\'with',
+					\'yield']
```

Comment:
Dude... I'm the author and only user of the omnifunc, this change will just make it easier for ***you*** and not the actual maintainer of the plugin. The order was the one from the grammar which was and can also be copied and `:s///` easily, it's easier for me/maintainer to update it from the grammar than from your custom alphabetized keyword list.

Even if you try to make future diffs clearer, the result is hiding all the changes from this PR and I fear that you removed another useful "keyword/annotation" but I can't see it.